rollup/es.config.js

```diff
@@ -1,16 +1,46 @@
 import {nodeResolve} from '@rollup/plugin-node-resolve';
-import {terser} from 'rollup-plugin-terser';
+import commonjs from '@rollup/plugin-commonjs';
+import json from '@rollup/plugin-json';
 
 export default {
   input: './esm/index.js',
   plugins: [
+    shims(),
     nodeResolve(),
-    terser()
+    commonjs(),
+    json()
   ],
   output: {
-    exports: 'named',
     file: './es.js',
-    format: 'iife',
-    name: 'linkedom'
+    format: 'esm'
   }
 };
+
+function shims() {
+  return {
+    resolveId(specifier) {
+      if(specifier === 'perf_hooks') {
+        return 'shim:perf_hooks';
+      }
+      if(specifier.endsWith('canvas.cjs')) {
+        return 'shim:canvas';
+      }
+    },
+    load(id) {
+      switch(id) {
+        case 'shim:perf_hooks': {
+          return `
+            export const performance = globalThis.performance;
+          `;
+        }
+        case 'shim:canvas': {
+          return `
+            export default {
+              createCanvas: () => {}
```

Comment:
maybe we can use the callback here, so that it won't fully break when/if used.

```js
        return `
          class Canvas {
              constructor(width, height) {
                this.width = width;
                this.height = height;
              }
              getContext() { return null; }
              toDataURL() { return ''; }
            }
            export default (width, height) => new Canvas(width, height);
          `;
```

This is basically the same fallback used in `canvas.cjs`.